### DISCUSSION TITLE
Local cache of PCHs

### DISCUSSION THREAD

**Discussion by wodin** [jank-lang/jank#21] [Issue Triage] [answered]
> One issue which may arise here is that the user running jank for the
> first time will need perms to write to where jank is installed. If this
> becomes an issue, we can add support for saving this someplace local,
> like ~/.jank as well.

It might be a good idea to use the [XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) to avoid creating a new top level hidden directory in the user's home directory:

> **$XDG_CACHE_HOME** defines the base directory relative to which user-specific non-essential data files should be stored. If $XDG_CACHE_HOME is either not set or empty, a default equal to **$HOME/.cache** should be used.

**Comment by jeaye** [comment]
Yep, great call out. That's a much better approach. :)

**Reply by jeaye** [reply, marked as answer]
We do this now, ever since b1d1f2c2156d82a434d490cb94ee7dea4f36c613.
